polls/management/commands/cleanup.py

```diff
@@ -13,16 +14,19 @@ def handle(self, *args, **kwargs):
         with open('polls/fixtures/initial_data.json') as fp:
             initial_data = json.load(fp)
 
-        initial_questions = filter(lambda m: m['model'] == 'polls.question', initial_data)
+        initial_questions = filter(
+            lambda m: m['model'] == 'polls.question', initial_data
+        )
         initial_question_pks = map(lambda m: m['pk'], initial_questions)
         one_hour_ago = timezone.now() - timedelta(hours=1)
-        qs = Question.objects.exclude(id__in=initial_question_pks).filter(published_at__lt=one_hour_ago)
+        qs = Question.objects.exclude(id__in=initial_question_pks).filter(
+            published_at__lt=one_hour_ago
+        )
 
         print('Deleting {} questions'.format(qs.count()))
         qs.delete()
 
-        qa = Vote.objects.all()
+        qs = Vote.objects.all()
```

Comment:
linting exposed that `qa` was an unused variable exposing the bug where deleting votes (next two lines) was a no-op and not doing the desired behaviour.

Reply:
Should we split this into a separate commit for clarity?

Reply:
Yeah you're right, I was taking shortcuts. Addressed in 122ec55